src/cpu/paging.cpp

```diff
@@ -214,16 +214,16 @@ static inline bool InitPageCheckPresence_CheckOnly(PhysPt lin_addr,bool writing,
 	const auto d_index=lin_page >> 10;
 	const auto t_index=lin_page & 0x3ff;
 	const auto table_addr=(paging.base.page<<12)+d_index*4;
-	table.load=phys_readd(table_addr);
-	if (!table.block.p) {
+	table.set(phys_readd(table_addr));
+	if (!table.p) {
 		paging.cr2=lin_addr;
 		cpu.exception.which=EXCEPTION_PF;
 		cpu.exception.error=(writing?0x02:0x00) | (((cpu.cpl&cpu.mpl)==0)?0x00:0x04);
 		return false;
 	}
-	const auto entry_addr=(table.block.base<<12)+t_index*4;
-	entry.load=phys_readd(entry_addr);
-	if (!entry.block.p) {
+	const auto entry_addr=(table.base<<12)+t_index*4;
```

Comment:
Suggest a follow-up `./scripts/format-commit.sh` commit that formats these changes.